static/js/controllers/inbox.js

```diff
@@ -799,16 +798,23 @@ var feedback = require('../modules/feedback'),
         if (window.applicationCache.status === window.applicationCache.UPDATEREADY) {
           showUpdateReady();
         }
-        WatchDesignDoc(function() {
-          // if the manifest hasn't changed, prompt user to reload settings
-          window.applicationCache.addEventListener('noupdate', showUpdateReady);
-          // check if the manifest has changed. if it has, download and prompt
-          try {
-            window.applicationCache.update();
-          } catch(e) {
-            // chrome incognito mode active
-            $log.error('Error updating the appcache.', e);
-            showUpdateReady();
+        Changes({
```

Comment:
'Scuse my ignorance, but does this mean we are still requesting two separate changes feeds?  Or there is a single feed, which triggers two callbacks (one for `medic-client` ddoc, and the other for all the other docs)?